integrations/standalone/tests/integration/parts/call.ts

```diff
@@ -17,7 +17,7 @@ export class CallTester implements PartTest {
   }
   async fill(page: Page) {
     await ComboboxUtil.select(page, this.selectValue, this.selectLabel);
-    await TableUtil.fillExpression(page, 2, '"test"');
```

Comment:
I general we talk about the expression and not the code in mapping tables. 
I find the old naming better.

Reply:
ok i change it back 👍